scripts/githooks/prepare-commit-msg

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+readonly version=v0.0.1
+readonly project_name=go-sdk
+readonly commit_tags=feat:\|fix:\|style:\|refactor:\|test:\|docs:\|chore:\|build:\|ci:\|perf:\|metric:\|misc:
```

Comment:
I've added 'misc' to the list of valid commands. Giving us a way to circumvent the validation if we need to for some reason. Undecided on whether to keep it in or not.